services/minespace-web/src/components/Forms/noticeOfDeparture/AddNoticeOfDepartureForm.js

```diff
@@ -1,13 +1,16 @@
 import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
-import { Field, reduxForm } from "redux-form";
+import { change, Field, reduxForm } from "redux-form";
 import { Button, Col, Popconfirm, Row, Typography } from "antd";
 import { Form } from "@ant-design/compatible";
 import { maxLength, required, requiredList, validateSelectOptions } from "@common/utils/Validate";
 import { resetForm } from "@common/utils/helpers";
+import { DOCUMENT, EXCEL } from "@/constants/fileTypes";
 import { renderConfig } from "@/components/common/config";
 import * as FORM from "@/constants/forms";
 import CustomPropTypes from "@/customPropTypes";
+import NoticeOfDepartureFileUpload from "@/components/Forms/noticeOfDeparture/NoticeOfDepartureFileUpload";
+import { NOTICE_OF_DEPARTURE_DOCUMENT_TYPE } from "../../../../common/constants/strings";
```

Comment:
shouldn't this be `@common/constants/strings` right?

Reply:
Good catch.  The IDE sometimes grabs them from their path automatically and adds those imports.  Didn't see it happen!